tasks/nunjucks.js

```diff
@@ -22,6 +22,9 @@ module.exports = function(grunt) {
         }
 
         var envOptions = { watch: false };
+        if (typeof options.autoescape === 'boolean') {
+            envOptions.autoescape = options.autoescape;
+        }
```

Comment:
Yes, but in such case we will override default Nunjucks' value.

For now it's `true` too, but if it will change in future — we'll have to fix it here too.

If it's ok for you, I will make commit with your code